packages/ember-test-app/app/templates/marketing.hbs

```diff
@@ -198,5 +198,60 @@
         </div>
       </div>
     </form>
+
+    <Mktg::ServicePricing
+      @label="Fiber"
+      @type="The Gig"
```

Comment:
I don't see the type property listed in the component. Should this be `package`?